code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -417,13 +419,15 @@
 	interface_desc = "Leave your mark."
 	engage_string = "Toggle stamp type"
 	usable = 1
-	var/iastamp
-	var/deniedstamp
+	var/obj/iastamp			//Theese were just vars, but any device would need to be an object
+	var/obj/deniedstamp //Stops assigning non-objects to theese vars, which probably would break quite a bit.
```

Comment:
BYOND doesn't give a flying fuck about typing on variables, it will assign a number to a `var/datum/x` variable happily.